### DISCUSSION TITLE
up主给的相关法律其实是坑！！

### DISCUSSION THREAD

**Discussion by jingzz1** [WeNeedHome/SummaryOfLoanSuspension#852] [General]
[相关法律与审判案例支持](https://github.com/WeNeedHome/SummaryOfLoanSuspension/blob/main/%E6%B3%95%E7%90%86%E4%BE%9D%E6%8D%AE/%E7%9B%B8%E5%85%B3%E6%B3%95%E5%BE%8B%E4%B8%8E%E5%AE%A1%E5%88%A4%E6%A1%88%E4%BE%8B%E6%94%AF%E6%8C%81.md)

《关于审理商品房买卖合同纠纷案件适用法律若干问题的解释》从2003年到现在已经20年了，可是国内依旧有人在住烂尾楼，而判决胜诉 这么多年也只有那两例，这就说明这份解释并不适合于烂尾楼。
举几个例子：
```
第十九条 商品房买卖合同约定或者《城市房地产开发经营管理条例》第三十三条规定的办理房屋所有权登记的期限届满后超过一年，由于出卖人的原因，导致买受人无法办理房屋所有权登记，买受人请求解除合同和赔偿损失的，应予支持。
第二十四条 因商品房买卖合同被确认无效或者被撤销、解除，致使商品房担保贷款合同的目的无法实现，当事人请求解除商品房担保贷款合同的，应予支持。
第二十五条 ……商品房买卖合同被确认无效或者被撤销、解除后，商品房担保贷款合同也被解除的、出卖人应当将收受的购房和购房款的本金及利息分别返还担保权人和买受人。
```
交房期限满一年还没交房的，可以取消购房合同，然后开发商赔偿买受人（业主）购房款和利息损失，返还银行购房款，三者互不相欠，看着是不是很激动，然而这里给业主埋了好几个坑，随便一个都可能让业主钱房两空，首先，购房合同取消后，业主和房产就没有任何关系了，以后有其他开发商接手，房子建好也和业主没有关系。
然后业主和开发商的关系转变成债务关系，还款顺序排到最后，如果开发商有钱还好，但如果开发商没钱呢？法律可没规定开发商的赔偿时间，如果开发商一直拖着不赔偿，拖个十几二十年的，那生活还要不要继续？当然，业主可以和开发商约定赔偿的最后期限，然而期限到了也不代表能拿到钱，就算开发商清盘了，业主和开发商是债务关系，排到业主的时候开发商还有钱吗？
还有，取消的只是“购房合同”，“借款合同”还没取消的，所以开发商没赔偿的话，业主还要继续还房贷的。

```
第二十四条因商品房买卖合同被确认无效或者被撤销、解除，致使商品房担保贷款合同的目的无法实现，当事人请求解除商品房担保贷款合同的，应予支持。

第二十五条以担保贷款为付款方式的商品房买卖合同的当事人一方请求确认商品房买卖合同无效或者撤销、解除合同的，如果担保权人作为有独立请求权第三人提出诉讼请求，应当与商品房担保贷款合同纠纷合并审理;未提出诉讼请求的，仅处理商品房买卖合同纠纷。担保权人就商品房担保贷款合同纠纷另行起诉的，可以与商品房买卖合同纠纷合并审理。商品房买卖合同被确认无效或者被撤销、解除后，商品房担保贷款合同也被解除的、出卖人应当将收受的购房贷款和购房款的本金及利息分别返还担保权人和买受人。
```
这两条说的是，“购房合同”取消后，当事人可以申请取消“担保贷款合同”，先申请取消“担保合同”，再取消“借款合同”，“担保合同”的当事人是谁？抵押物是房子，业主的房产证还没拿到，所以这里的当事人是担保权人(银行)和开发商，想取消的话只能让银行起诉开发商，如果开发商有钱的话，银行当然不介意,但如果开发商没钱呢？企业有限责任，个人无限责任，你猜银行希望这个债由谁来背？

“抵押合同”没取消的话，房贷就得继续还，一般是要银行有过失才被取消，胜诉的那两个案例都是一波三折，嘉兴那个案子是因为法院认定银行有霸王条款，现实中真要上了法庭，基本上都是银行打赢。

**Comment by richardo876** [comment]
你的意思是要有银行过错的证据吗？ 如果能有银行违规放贷的证据是不是一个突破点？

**Reply by jingzz1** [reply]
是，赢的那两个都是银行有过错，但能赢的机率实在太低了，二十多年就那两个，而且输了代价太大，银行输了赔点钱，业主输了房子没了，房贷要还，赔偿银行损失，可能还要补充其他抵押物。

**Reply by Publius-thebard** [reply]
其实很多，只是引发关注的只有这两个

**Comment by Accelerator6666** [comment]
这里要引入两个部门的责任，让它们去斗。1，房管局对监管账户资金的审批责任。2，银行对于买家抵押贷款的款项去向问题。

**Comment by Publius-thebard** [comment]
最高院的那个案子才是真正值得研究的判例，那个案子基本定了这几年的基调。但要注意的是烂尾不是你说了算，延期交付和解除购房合同不是等价的，需要个案判断。但确定解除购房合同就一定可以解除借款合同，按最高院的说法就是合同目的落空而解除。担保合同作为从合同也解除。但结果是什么呢，就是你失去了首付和已还款，费心打了官司后合法停贷：因为开发商破产基本追不回贷款。而官司打不打得赢就看你是否能达到解除购房合同的地步，这是由法院判断的。

**Comment by Publius-thebard** [comment]
指出一下你的误解。“担保贷款合同”，指的是有担保条款的贷款合同，这两个是一起取消的。案例里的银行过失很简单的，就是银行撇清所有责任的格式条款，大多数合同是符合这个条件的
